### DISCUSSION TITLE
LDAP search on custom attributes

### DISCUSSION THREAD

**Discussion by icanhazbeer** [grafana/grafana#61111] [General]
Hello,
Does Grafana support search filtering on LDAP custom attributes?
I would like to provide a search filter for only users that have certain extension attributes.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
